Makefile

```diff
@@ -204,8 +206,9 @@ bundle: static-manifests gen-semver kustomize ## Generate bundle manifests and m
 	operator-sdk bundle validate ./bundle
 
 .PHONY: bundle-build
-bundle-build: gen-semver build-base-image ## Build the bundle image.
-	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) --build-arg BASEIMAGE=$(BASEIMAGE) .
+bundle-build: gen-semver download-csm-common ## Build the bundle image.
+	$(eval include csm-common.mk)
+	docker build --pull -f bundle.Dockerfile -t $(BUNDLE_IMG) --build-arg BASEIMAGE=$(CSM_BASEIMAGE) .
```

Comment:
Does this require docker? `podman` is used elsewhere.

Reply:
I did not want to change the builder as part of this PR. The original code uses docker.